src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,545 @@
+            upadePotStackAmountInHandBigBlindPlayer(amountToCallBigBlind(player1, player2), player1);
+            printPot();
+            makeFlop();
+        } else if (decision2 == 2) {
+            makeRaiseBigBlind(player1, player2);
+            preFlopInPositionPlayerCantCheck(player1, player2);
+        } else {
+            System.out.println(player1.getNamePlayer() + " folds");
+            System.out.println(player2.getNamePlayer() + " wins " + gameState.getPot() + "$");
+        }
+    }
+
+    public void preFlopInPositionPlayerCantCheck(Player player1, Player player2) {
+        System.out.println(player2.getNamePlayer() + " decyduje");
+        printPot();
+        howMuchToCallSmallBlind(player1, player2);
+        System.out.println("1 = call 2 = raise 3 = fold");
+        Scanner scanner = new Scanner(System.in);
+        int decision2 = scanner.nextInt();
+        if (decision2 == 1) {
+            if (amountToCallSmallBlind(player1, player2) >= player2.getStackPlayer()) {
+                gameState.updatePot(amountToCallSmallBlind(player1, player2));
+                System.out.println(player2.getNamePlayer() + " All in " + (amountToCallSmallBlind(player1, player2) + player2.getAmountInHand()) + "&");
+                System.out.println("pot wynosi " + gameState.getPot() + "$");
+                equityEvaluator.calculateEquity(result.getHand1(), result.getHand2(), gameResult);
+            } else {
+                gameState.updatePot(amountToCallSmallBlind(player1, player2));
+                System.out.println(player2.getNamePlayer() + " calls " + amountToCallSmallBlind(player1, player2) + "$");
+                player2.updateStackPlayer(amountToCallSmallBlind(player1, player2));
+                player2.updateAmountInHand(amountToCallSmallBlind(player1, player2));
+                printPot();
+                makeFlop();
+            }
+        } else if (decision2 == 2) {
+            makeRaiseSmallBlind(player1, player2, amountToCallSmallBlind(player1, player2));
+            preFlopOutPositionPlayerGotRaise(player1, player2);
+        } else {
+            System.out.println(player2.getNamePlayer() + " folds");
+            System.out.println(player1.getNamePlayer() + " wins " + gameState.getPot() + "$");
+        }
+    }
+
+    public void howMuchToCallBigBlind(Player player1, Player player2) {
+        System.out.println(amountToCallBigBlind(player1, player2) + "$ to call");
+    }
+
+    public void howMuchToCallSmallBlind(Player player1, Player player2) {
+        System.out.println(amountToCallSmallBlind(player1, player2) + "$ to call");
+    }
+
+    public double amountToCallBigBlind(Player player1, Player player2) {
+        return player2.getAmountInHand() - player1.getAmountInHand();
+
+    }
+
+    public double amountToCallSmallBlind(Player player1, Player player2) {
+        return player1.getAmountInHand() - player2.getAmountInHand();
+    }
```

Comment:
wydaje mi sie ze na spokojnie dalo by sie to zrobic 1 funckja np amountToCall(Player player1, Player player2) i w kodzie wywolywac to raz amountToCall(p1, p2) a raz amountToCall(p2, p1) a jesli tak sie nie da to na pewno fukcja amountToCallBigBlind() i  amountToCallSmallBlind() moze byc tak zamieniona i mixowac tymi argumetnami w howMuchToCallBigBlind howMuchToCallSmallBlind (i zmienic ich nazwy na amountToCallBigBlind()/amountToCallSamllBlind())